packages/flutter/test/material/mergeable_material_test.dart

```diff
@@ -5,6 +5,8 @@
 import 'package:flutter/material.dart';
 import 'package:flutter_test/flutter_test.dart';
 
+import '../rendering/mock_canvas.dart';
```

Comment:
My thinking is to do that once we have a few more clients. We probably want to dress up the API a bit more, and it will be more obvious how to do that when we have a few clients.